CODE_STYLE.md

```diff
@@ -2,3 +2,14 @@
 
 * [Ruby](https://github.com/bbatsov/ruby-style-guide)
 * [Objective-C](https://github.com/hyperoslo/objective-c-style-guide)
+* [JavaScript](https://github.com/hyperoslo/eslint-config)
+
+## Stylus
+
+We have not yet chosen a style guide for Stylus, other than to decide that:
+
+* We *don't* use `;`.
+* We *do* use `{` and `}`.
```

Comment:
Yeah, I'm also generally for the curly braces as I find it it makes nested selectors more readable, but I'm mostly for finding a standard we can all live with, so :+1: for removing them. :wink: